src/ios/CodePush.m

```diff
@@ -314,6 +314,10 @@ - (void)saveFailedUpdate:(NSString *)packageHash {
     NSMutableArray* failedUpdates = [preferences objectForKey:FailedUpdatesKey];
```

Comment:
Since this **not** actually an NSMutableArray, just for added clarity, should we store [preferences objectForKey:FailedUpdatesKey] into a normal immutable NSArray variable first, then call ```mutableCopy``` on it?